src/web/components/AdSlot.tsx

```diff
@@ -267,52 +295,55 @@ export const AdSlot: React.FC<Props> = ({ position, display }) => {
 		case 'top-above-nav': {
 			const adSlotAboveNav = css`
 				margin: 0 auto;
-				height: 151px;
+				min-height: 108px;
```

Comment:
Previously this calculated height as:
`smallest ad height (90px) + 2 * padding  (2 * 18px) + ad label (24px) = 150px`

`padding-top` [previous PR](https://github.com/guardian/dotcom-rendering/pull/2668) and the ad label (this PR) no longer exist within `top-above-nav`.

Calculation is now:
`smallest ad height (90px) + 1 * padding  (1 * 18px) = 108px`